src/app/app.module.ts

```diff
@@ -48,7 +50,7 @@ import { ReactiveFormsModule, FormsModule } from '@angular/forms';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    AngularFireModule,
+    AngularFireModule.initializeApp(environment.firebase),
```

Comment:
Please, not override the config for initialization. look src/main.ts and https://github.com/angular/angularfire/issues/2360#issuecomment-607916972

Reply:
Done